maat-court-data-api/src/test/java/gov/uk/courtdata/correspondence/controller/CorrespondenceStateControllerTest.java

```diff
@@ -0,0 +1,129 @@
+package gov.uk.courtdata.correspondence.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import gov.uk.courtdata.correspondence.dto.CorrespondenceStateDTO;
+import gov.uk.courtdata.correspondence.service.CorrespondenceStateService;
+import gov.uk.courtdata.exception.RequestedObjectNotFoundException;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import static gov.uk.courtdata.builder.TestModelDataBuilder.*;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@ExtendWith(SpringExtension.class)
+@WebMvcTest(CorrespondenceStateController.class)
+public class CorrespondenceStateControllerTest {
+
+    private static final String ENDPOINT_URL = "/api/internal/v1/assessment/correspondence-state";
+    private static int INVALID_REP_ID = 1235;
```

Comment:
`INVALID_REP_ID` should also be declared `final`

Reply:
yes I agree, missed it